main.js

```diff
@@ -0,0 +1,66 @@
+const sectionSpeakers = document.querySelector(".speaker-wrap");
+
+// SPEAKERS INFORMATION //
+
+const speakersInfo = [
+    {
+        photo: './images/speakers/Riccardo_Accolla.png',
+        alt: 'Riccardo Acollas portrait',
+        name: 'Riccardo Acolla',
+        title: 'Director of Digital Food Science at Ripe Technology',
+        career: 'He has scientific and technical experience in flavor and nutrition, as well as knowledge in the process of developing ingredients and products.',
+    },
```

Comment:
- [ ] Great job generating the speakers section dynamically from JavaScript 💯 In the mobile view of the homepage, kindly display only 2 speakers together with a `more` button so as to follow the required design 👍 
- [ ] (OPTIONAL) You can also implement the functionality of the `more` button in the mobile view. When a user clicks on it, the rest of the speakers should then be displayed. This will improve the UX of your site 💯 


- Required design:


![Screenshot 2022-06-30 190829](https://user-images.githubusercontent.com/85113567/176748546-63fc156b-9dde-4741-bfde-23af16695997.png)
- Your design:
![Screenshot 2022-06-30 190525](https://user-images.githubusercontent.com/85113567/176748931-24f5d6fd-da77-4cea-89c5-2c130d5085ed.png)